features/step_definitions/user_profile_form_steps.rb

```diff
@@ -42,7 +42,7 @@
 
 When(/^there is no (.+) associated with my account$/) do |attribute|
   attribute = attribute.parameterize('_')
-  @user.update_attribute(attribute, nil)
+  @user.update(attribute.to_sym => '')
```

Comment:
I prefer `update` to `update_attribute` bc it runs validations. See http://www.davidverhasselt.com/set-attributes-in-activerecord